packages/info/doctor/mixin.core.js

```diff
@@ -34,9 +34,14 @@ class DoctorMixin extends Mixin {
     );
   }
 
-  diagnose({ validateConfig, detectDuplicatePackages }) {
+  diagnose({ validateConfig, detectDuplicatePackages, pushError }) {
     validateConfig();
-    detectDuplicatePackages('hops-*');
+    if (detectDuplicatePackages('hops-*')) {
```

Comment:
We should also check for `*-hops*` here, to catch `jest-preset-hops` and `jest-environment-hops`.